docs/docs/waiting.md

```diff
@@ -40,6 +40,6 @@ There are several reasons why your app or pod may fail to deploy. Some possibili
   ```
 
 - Your application, application group, or pod definition is otherwise badly configured.
-  You can use the [marathon-validate](https://github.com/dcos-labs/marathon-validate) script to validate an app or group definition locally, before you deploy it to DC/OS.
+  You can use the [marathon-validate](https://github.com/dcos-labs/marathon-validate) script to validate an app or group definition locally, before you deploy it to DC/OS. **Warning: This tool is not up-to-date with the current marathon version**.
```

Comment:
if it is not up to date I would be inclined to remove this recommendation from docs completely...